democracy/views/utils.py

```diff
@@ -97,3 +99,56 @@ def filter_by_hearing_visible(queryset, request, hearing_lookup='hearing'):
         filters['%s__published' % hearing_lookup] = True
 
     return queryset.filter(**filters)
+
+
+class JSONPKRelatedField(PrimaryKeyRelatedField):
+    """
+    Support of showing and saving of expanded JSON nesting or just a resource
+    ID.
+    Serializing is controlled by query string param 'expand', deserialization
+    by format of JSON given.
+    Default serializing is expand=false.
+    """
+
+    invalid_json_error = _('Incorrect JSON. Expected JSON, received %s.')
+    missing_id_error = _('The primary key is missing. Expected {"id": id, ...}, received %s.')
+
+    def __init__(self, *args, **kwargs):
+        self.related_serializer = kwargs.pop('serializer', None)
+        self.expanded = kwargs.pop('expanded', False)
```

Comment:
Maybe `self.expanded = bool(kwargs.pop('expanded', False))` to ensure booliness.